docs-2.0/7.data-security/4.ssl.md

```diff
@@ -0,0 +1,64 @@
+# SSL encryption
+
+Nebula Graph supports transmission with SSL encryption between the client, Graph service, Meta service, and Storage service. This topic describes how to enable SSL encryption.
+
+## Precaution
+
+Enabling SSL encryption will slightly affect performance.
+
+## Parameters
+
+| Parameter | Default value | Description |
+| ------------------ | ------- | ------------------------------|
+| `cert_path`        | -       | The path to the pem certification. |
+| `key_path`         | -       | The path to the key certification. |
+| `password_path`    | -       | The path to the password file certification. |
+| `ca_path`          | -       | The path to the trusted CA file. |
+| `enable_ssl`       | `false`   | Whether to enable SSL encryption. |
+| `enable_graph_ssl` | `false`   | Whether to enable SSL encryption in Graph service only. |
```

Comment:
Graph services are distributed in a cluster. will it be better to use services or a Graph service here?